dataRetriever/blockchain/baseBlockchain.go

```diff
@@ -100,3 +102,20 @@ func (bbc *baseBlockChain) GetFinalBlockInfo() (uint64, []byte, []byte) {
 
 	return nonce, hash, rootHash
 }
+
+// SetCurrentAggregatedSignatureAndBitmap sets the current aggregated signature and its validator's public keys bitmap
+func (bbc *baseBlockChain) SetCurrentAggregatedSignatureAndBitmap(signature []byte, pubKeysBitmap []byte) {
+	bbc.mut.Lock()
+	defer bbc.mut.Unlock()
+
+	bbc.currentAggregatedSignature = signature
+	bbc.currentPubKeysBitmap = pubKeysBitmap
+}
+
+// GetCurrentAggregatedSignatureAndBitmap returns the current aggregated signature and its validator's public keys bitmap for the current block
+func (bbc *baseBlockChain) GetCurrentAggregatedSignatureAndBitmap() ([]byte, []byte) {
```

Comment:
I would find another way to return both byte slices. Reason: I can easily swap the return order on L120 and cause bugs that very hard to find

Reply:
updated

interface update here: https://github.com/multiversx/mx-chain-core-go/pull/282

Reply:
yup, better :+1: